.github/workflows/ci.yml

```diff
@@ -210,6 +210,8 @@ jobs:
             - { target: thumbv7m-none-eabi,               os: ubuntu-latest,                    std: 1 }
             - { target: cross,                            os: ubuntu-latest }
             - { target: zig,                              os: ubuntu-latest }
+            - { target: aarch64-unknown-linux-gnu, sub: centos, os: ubuntu-latest, dylib: 1, std: 1, run: 1, runners: qemu-user qemu-system }
```

Comment:
This should have cpp support. Also, I'm not sure even if this supports C. See comment below for why.